lib/formatters/stringFormatter.mjs

```diff
@@ -249,7 +251,12 @@ export default function stringFormatter(results, returnValue) {
 		);
 
 		for (const warning of result.warnings) {
-			calcSeverityCounts(warning.severity, counts);
+			calcSeverityCounts(warning.severity, resultCounts);
+			calcMaybeFixableCounts(
+				warning.severity,
+				returnValue.ruleMetadata?.[warning.rule]?.fixable ?? false,
+				fixableCounts,
+			);
```

Comment:
[suggest] How about reusing `calcSeverityCounts` like this? If so, `calcMaybeFixableCounts` seems unneeded.

```suggestion
			const fixable = returnValue.ruleMetadata?.[warning.rule]?.fixable;

			if (fixable === true) {
				calcSeverityCounts(warning.severity, fixableCounts);
			}
```

Reply:
That's much better. Updated!